doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/proc-guided-decision-tables-columns-sort.adoc

```diff
@@ -0,0 +1,11 @@
+[id='proc-guided-decision-tables-columns-sort_{context}']
+= Sorting column values in guided decision tables
+
+You can sort the values in columns that you created in a guided decision table.
+
+.Prerequisites
+* You have created the required columns in a guided decision table.
```

Comment:
If you keep "you", please still remove "have". The style guide discourages the perfect tenses.

Reply:
Done, removed have.